platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/DataFileReader.java

```diff
@@ -376,6 +379,11 @@ private ByteBuffer read(final long byteOffsetInFile, final int bytesToRead) thro
         for (int retries = 3; retries > 0; retries--) {
             final int fcIndex = leaseFileChannel();
             final FileChannel fileChannel = fileChannels.get(fcIndex);
+            if (fileChannel == null) {
```

Comment:
Do we need to check if the file channel is actually open here (e.g. ` || !filChannel.isOpen()`)?

Reply:
I don't think we need this condition. If this code even encounters a closed channel, it processes the exception here:
```
            } catch (final ClosedChannelException e) {
                // This exception may be thrown, if the channel was closed, because a different
                // thread reading from the channel was interrupted. Re-create the file channel
                // and retry
                reopenFileChannel(fcIndex, fileChannel);
            } 
```